src/components/CarItem.js

```diff
@@ -0,0 +1,70 @@
+import { CardActionArea } from '@mui/material';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Typography from '@mui/material/Typography';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+// import CardActionsContainer from './CardActions';
```

Comment:
- [ ] Please, kindly remove this old piece of commented code to improve the cleanliness of the repo 💪